netstandard/ref/System.Runtime.CompilerServices.cs

```diff
@@ -286,6 +286,12 @@ public partial interface IRuntimeVariables
     public static partial class IsBoxed
     {
     }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(8))]
```

Comment:
FYI - @ericstj recently fixed cci extensions to output the attribute values now https://github.com/dotnet/buildtools/pull/2182/commits/1dc2ff35acf25819f836310a8510808a461e8e3a.

Reply:
Sweet. I'll do another pass to make the reference assemblies legible.